### ISSUE TITLE
Import-Oberfläche instabil

### ISSUE THREAD

**Issue by michaelgiesler** [typearea/dembelo#326]
Derzeit hilft nur vieles Neuladen der Seite, um eine Twine-Datei importieren zu können. Die Probleme müssen analysiert und behoben werden.

**Comment by SarahSauer** [comment]
Ich spiele gerade etwas im Backend herum. Ist das der Error 41 oder bin ich über was Neues gestolpert?

==> Fehler: Error #41: 'Attribute without value' occurred while parsing the Twine archive file '/var/www/demo.waszulesen.de/htdocs/www/app/../uploads/twine/58da8f4682bf9c06518868ca/58da8f9a82bf9c16518868ca' in line 3, character 127 (at byte index 253).

(Zum Kontext: Ich hab versucht, eine Twine Datei zu importieren)

**Comment by michaelgiesler** [comment]
Nein, das ist was anderes. in diesem Issue soll es um die Oberfläche gehen. Deine Fehlermeldung ist ein Problem im Twine-HTML und ist mir letzte Woche bereits begegnet. Issue #339.
